test/acceptor_SUITE.erl

```diff
@@ -1373,3 +1428,12 @@ do_os_supports_reuseport() ->
 		{{unix, linux}, {3, Minor, _}} when Minor >= 9 -> true;
 		_ -> false
 	end.
+
+do_os_supports_local_sockets() ->
+	case os:type() of
+		{unix, _} -> true;
+		_ -> false
+	end.
+
+do_tempname() ->
+	filename:flatten(["/tmp/", ?MODULE, $_, [$0 + rand:uniform(9) || _ <- lists:seq(1, 10)], ".sock"]).
```

Comment:
I can guess you are not too happy with this, but I need to create a socket file somewhere. The functions using this one work under the assumption they are on unix, therefore the static `/tmp` should work. The random part of the file name is there to avoid clashes with existing files, ie when a previous run of the tests in question failed and didn't delete the file.

Reply:
Just use `lists:reverse(tl(lists:reverse(os:cmd("mktemp"))))` it's available everywhere.

Reply:
That won't do, as the file will be created by mktemp. The file given to the listen call must not exist at that time, the call must be able to create it. I could use mktemp with the -u (dry run) switch that doesn't create a file. The man page says that this is unsafe, but I guess safe enough for testing purposes. What do you think?

Reply:
Sure.